src/Components/Publishing/Sections/FullscreenViewer/Caption.tsx

```diff
@@ -1,123 +1,83 @@
-import Colors from "Assets/Colors"
-import { unica } from "Assets/Fonts"
-import { pMedia } from "Components/Helpers"
-import PropTypes from "prop-types"
+import { Box, Clickable, HTML, Text } from "@artsy/palette"
 import React from "react"
-import styled, { StyledFunction } from "styled-components"
-import { ArtworkCaption } from "../ArtworkCaption"
+import PropTypes from "prop-types"
+import { ArtworkCaption } from "./ArtworkCaption"
+import { Media } from "Utils/Responsive"
 
-interface CaptionProps extends React.HTMLProps<HTMLDivElement> {
-  section?: any
+interface CaptionProps {
+  section?: any // TODO: What am I?
```

Comment:
`SectionData`: https://github.com/artsy/reaction/blob/7bf92155d83cad72b6c24582bb79879294b5511e/src/Components/Publishing/Typings.ts#L129

Reply:
I think the section naming might actually be misleading, and this is actually representative of a single item in an image array, `article.sections[i].images` - https://github.com/artsy/reaction/blob/7bf92155d83cad72b6c24582bb79879294b5511e/src/Components/Publishing/Typings.ts#L144

Reply:
This could still be typed as `ImageData`, otherwise looks ok.